docs/docs/reference.md

```diff
@@ -1,24 +1,3 @@
 ---
-title: Reference
-id: reference
-layout: docs
-description: Secretless Broker Documentation
-permalink: docs/reference.html
+redirect_to:
```

Comment:
unsure if we want to keep this because there are no `references` in the new secretless docs on flare